fastlane_core/lib/fastlane_core/print_table.rb

```diff
@@ -24,7 +24,14 @@ def print_values(config: nil, title: nil, hide_keys: [], mask_keys: [])
         rows = self.collect_rows(options: options, hide_keys: hide_keys.map(&:to_s), mask_keys: mask_keys.map(&:to_s), prefix: '')
 
         params = {}
-        params[:rows] = limit_row_size(rows)
+        transform = false if FastlaneCore::Env.truthy?("FL_SKIP_TABLE_TRANSFORM")
```

Comment:
rather than overloading this, I'd maybe extract out a `transform?` method that is `!FastlaneCore::Env.truthy?...` and have the if statement be `if transform? ...` - this means you don't need to trace through mutating state.